examples/stable-diffusion/text_to_video_generation.py

```diff
@@ -52,6 +52,13 @@ def main():
         type=str,
         help="Path to pre-trained model",
     )
+    parser.add_argument(
+        "--pipeline_type",
+        type=str,
+        nargs="*",
+        default="sdp",
+        help="pipeline type:sdp or cogvideoX",
```

Comment:
```suggestion
        default="stable_diffusion",
        help="pipeline type: stable_diffusion or cogvideoX",
```
I still find sdp not very explicit

Reply:
resolved